src/app/static/src/app/store/load/actions.ts

```diff
@@ -56,6 +57,15 @@ export const actions: ActionTree<LoadState, RootState> & LoadActions = {
         const files = objectContents.files;
         const savedState = objectContents.state;
 
+        const majorVersion = (s: string) => s ? s.split(".")[0] : null;
+        if (majorVersion(savedState.version) != majorVersion(currentHintVersion)) {
```

Comment:
Considering we've never incremented this version number and have no protocol for doing so probably semantic versioning is overkill here and just v1, v2 etc might suffice! But fine to leave this as is, maybe we can have a bit more of a think about this longer term - I don't think we do any release versioning of the docker image either? Would be nice to figure out a strategy that ties in with releasing...